spec/models/reviewable_history_spec.rb

```diff
@@ -3,11 +3,13 @@
 require 'rails_helper'
 
 RSpec.describe ReviewableHistory, type: :model do
-
   fab!(:user) { Fabricate(:user) }
   fab!(:admin) { Fabricate(:admin) }
   fab!(:moderator) { Fabricate(:moderator) }
 
+  it { is_expected.to define_enum_for(:status).with_values(%i[pending approved rejected ignored deleted]) }
```

Comment:
Not sure we need a spec that detailed. I'd rather we test all the items of the enum in "broader" tests instead.

Reply:
Ok, I’m used to spec everything in isolation and then have integration/acceptance tests on top of that :sweat_smile:  Shoulda matchers make it quite easy for the former :slightly_smiling_face: